module/VuFind/src/VuFind/Cover/Loader.php

```diff
@@ -301,7 +301,12 @@ protected function getImageSettingsFromLegacyArgs($args)
     protected function storeSanitizedSettings($settings)
     {
         $settings = array_merge($this->getDefaultSettings(), $settings);
-        $this->isbn = new ISBN($settings['isbn'] ?? '');
+        $this->isbns = array_map(
+            function ($isbn) {
+                return new ISBN($isbn);
+            },
+            $settings['isbns'] ?? []
```

Comment:
Similarly, for backward compatibility, maybe we should support the possibility of the legacy 'isbn' value still being used, like:

`$settings['isbns'] ?? (empty($settings['isbn']) ? [] : (array)$settings['isbn'])`

...just in case a custom record driver somewhere is still using the old setting.

Reply:
Done. Thanks for suggestions.